src/data-handling/Traveler.js

```diff
@@ -1,4 +1,28 @@
+import moment from 'moment';
+moment().format();
```

Comment:
I need to learn more about moment. What exactly is the '.format()' function doing here?